requirements/extra-pdf-image.in

```diff
@@ -6,4 +6,4 @@ pdfminer.six
 # NOTE(robinson) - See this issue here
 # https://github.com/facebookresearch/detectron2/issues/5010
 Pillow<10
-unstructured-inference==0.5.9
+unstructured-inference==0.5.10
```

Comment:
can we move the pin to `contraints.in` and change this line to `unstructured-inference`?

Reply:
Done!